scripts/qa_crowbarsetup.sh

```diff
@@ -4506,13 +4506,25 @@ function onadmin_prepare_crowbar_upgrade
         # move nodes to upgrade mode
         safely crowbar_api_request POST $crowbar_api /installer/upgrade/prepare.json
     else
-        if safely crowbarctl upgrade repocheck crowbar --format plain | grep "missing" ; then
-            complain 11 "Some repository is missing on admin server. Cannot upgrade."
-        fi
-        # FIXME: crowbarctl command can time out easily
-        # Do not use it until https://bugzilla.novell.com/show_bug.cgi?id=997293 is fixed
-        # safely crowbarctl upgrade prepare
-        safely crowbar_api_request POST $crowbar_api /installer/upgrade/prepare.json
+        safely crowbarctl upgrade prepare
+        wait_for 200 5 "grep current_step /var/lib/crowbar/upgrade/progress.yml | grep -v upgrade_prepare" "prepare step to finish"
+    fi
```

Comment:
Unrelated to this PR:

How is the user of the cloud supposed to do this?
Wouldn't it make sense to either wait in the API until all is done and then send the reply or to offer a second API endpoint to query if its done?

@MaximilianMeister Can we maybe add some message to stderr of crowbarctl that informs about the fact, that this process takes a while?